views/pages/AccountPage/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { useOutletContext, Link } from 'react-router-dom'
+import { Button } from '@mui/material'
+
+import api from '../../api'
+import { routes } from '../../routes/routes'
+import UserProfileForm from '../../forms/UserProfileForm'
+import AccountPageHeader from './AccountPageHeader'
+import { borderRadius, fontSize } from '../../../constants'
+import FileModel from '../../models/FileModel'
+
+import './AccountPage.css'
+
+const AccountPage = () => {
+  const { user, setUser, setNotification } = useOutletContext()
+  const { display_name, icon, iconFileName, mail, title } = user
+  const iconFile =
+    !!icon && !!iconFileName
+      ? FileModel.fromDataURL(icon, iconFileName)
+      : undefined
+
+  const onSubmit = async (userProfileValues) => {
+    try {
+      const { iconFile, ...formData } = userProfileValues
+      const icon = iconFile ? await FileModel.toDataURL(iconFile) : ''
+      const iconFileName = iconFile ? iconFile.name : ''
+
+      const updatedUser = await api.users.update(user.uid, {
+        ...formData,
+        icon,
+        iconFileName,
+      })
+
+      setUser(updatedUser)
+      setNotification(() => ({
+        open: true,
+        message: 'User has been updated.',
+      }))
+    } catch (error) {
+      setNotification({
+        open: true,
+        message: error.message,
+      })
+    }
+  }
+
+  return (
+    <div className="AccountPage">
+      <div className="AccountPageBanner">
+        <AccountPageHeader />
+      </div>
+      <Button
+        variant="outlined"
+        size="small"
+        fullWidth={false}
+        component={Link}
+        to={routes.previewProfile}
+        sx={{
+          maxWidth: '121px',
+          textDecoration: 'none',
+          textTransform: 'none',
+          fontSize: fontSize[14],
+          borderRadius: borderRadius[8],
+        }}
+      >
+        Preview Profile
+      </Button>
+      <UserProfileForm
+        initialValues={{
+          iconFile,
+          display_name,
+          mail,
+          title,
+          iconFileName: iconFileName || '',
```

Comment:
we don't need this value in the form.

Reply:
[0730f7e](https://github.com/AMP-SCZ/dpdash/pull/478/commits/0730f7e99359b47d86b078fb3e0ad15f74bb8fcb)